test/test-dump.js

```diff
@@ -151,3 +162,33 @@ describe('dump MySQL', function () {
     stats.size.should.greaterThan(1600)
   })
 })
+
+describe('dump Postgres', function () {
+  before(function * () {
```

Comment:
Add comments sometimes : here you can easily tell us what the code does : filling database with test data for testing